app/lib/view_model/logic/preference/IPreferenceAccess.dart

```diff
@@ -0,0 +1,67 @@
+
+import '../../repository/data_classes/filter/FilterPreferences.dart';
+import '../../repository/data_classes/mealplan/Canteen.dart';
+import '../../repository/data_classes/settings/MensaColorScheme.dart';
+import '../../repository/data_classes/settings/MealPlanFormat.dart';
+import '../../repository/data_classes/settings/PriceCategory.dart';
+
+/// This is an interface for accessing the preferences.
+abstract class IPreferenceAccess {
+  /// The device identifier is returned.
```

Comment:
The varying names "client identifier" and "device identifier" are a bit confusing.